config/locales/js-en.yml

```diff
@@ -500,6 +501,8 @@ en:
         days: "days"
         weeks: "weeks"
         months: "months"
+      date:
+        exact: "date"
```

Comment:
Personally, I do not see any need for these labels in the UI as the filter operator itself is already stating that these are either weeks or days, or in this case, a date. And, it also complicates things in the UI template. I will remove the label from that filter.